### PR TITLE
Added mpi and ccl samples for covariance

### PR DESCRIPTION
**Pull request by Alexandr-Solovev**

### CODE REVIEW

File: samples/oneapi/dpc/ccl/sources/cov_distr_ccl.cpp [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  auto comm = dal::preview::spmd::make_communicator<dal::preview::spmd::backend::ccl>(queue);
+  auto rank_id = comm.get_rank();
+  auto rank_count = comm.get_rank_count();
+
+  auto input_vec =
+      split_table_by_rows<float>(queue, data, rank_count);
+
+  const auto result = dal::preview::compute(comm, cov_desc, input_vec[rank_id]);
+  if (comm.get_rank() == 0) {
+    //std::cout << "Mean:\n" << result.get_means() << std::endl;
```
Comment (amgrigoriev) [inline samples/oneapi/dpc/ccl/sources/cov_distr_ccl.cpp:51]: Please set optional results in descriptor to report more statistics

Comment (amgrigoriev) [reply samples/oneapi/dpc/ccl/sources/cov_distr_ccl.cpp:51]: The same for MPI sample

**Review by KulikovNikita — APPROVED** [review]

**Review by makart19 — APPROVED** [review]

**Pull request merged**